environment.yml

```diff
@@ -0,0 +1,107 @@
+name: basics-of-data-analytics
+channels:
+  - defaults
+dependencies:
```

Comment:
Though I didn't have any issues while creating the environment, guess we can clean this to only have the main packages like numpy, pandas, notebook, matplotlib and their respective versions only (like in time-series-labs). 

The thing with creating an environment with a file is when there's a single error during installing, it halts the whole installation process. Having only the main packages avoids such issues.

Reply:
These are jupyter notebooks related packages so I include it in

Reply:
It looks much better now. Thanks @KianYang-Lee . 👍